tests/test_cli.py

```diff
@@ -1335,7 +1335,8 @@ def test_use_source_timestamp(tmpdir, cwd_tmpdir, python_notebook, capsys, forma
     assert "Updating the timestamp" not in capture.out
 
     dest_timestamp = test_ipynb.stat().mtime
-    assert src_timestamp == dest_timestamp
+    # on Mac OS the dest_timestamp is truncated at the microsecond (#790)
+    assert src_timestamp - 1e-6 <= dest_timestamp <= src_timestamp
```

Comment:
Since you use pytest, you could use [pytest.approx](https://doc.pytest.org/en/latest/reference/reference.html#pytest.approx) here.

Reply:
Oh that's right... I like very much `pytest.approx` indeed! But here maybe the micro-second truncation is a bit more specific than a numerical uncertainty? Anyway thanks for commenting!

Reply:
Fair point!